src/fetch_datasets.py

```diff
@@ -38,15 +40,17 @@ def download_backup():
             print('Downloading %s' % filename)
             urlretrieve(url, filepath)
 
-
-
-parser = OptionParser(usage='usage: %prog [options]')
-parser.add_option('--from-source',
-                  action='store_true',
-                  help='download files as provided by original sources')
-
-(options, args) = parser.parse_args()
-if options.from_source:
+parser = ArgumentParser()
+parser.add_argument("-sr", "--from_source",
+                    action="store_true",
+                    help="download files as provided by original sources")
+parser.add_argument("-am", "--from_amazon",
```

Comment:
This makes the script fairly incompatible with some other scripts and instructions of this repo that uses no arguments/options at all..